cookbook/os/os_config/basic.py

```diff
@@ -0,0 +1,89 @@
+from agno.agent import Agent
+from agno.db.postgres import PostgresDb
+from agno.models.openai import OpenAIChat
+from agno.os import AgentOS
+from agno.os.interfaces.slack import Slack
+from agno.os.interfaces.whatsapp import Whatsapp
+from agno.os.schema import AgentOSConfig, ChatConfig, DatabaseConfig
+from agno.team import Team
+from agno.workflow.step import Step
+from agno.workflow.workflow import Workflow
+
+# Setup the database
+db = PostgresDb(db_url="postgresql+psycopg://ai:ai@localhost:5532/ai", id="basic-db")
```

Comment:
Can we rather use an ID like `db-1001` to make it easier to follow (or maybe it is just me)